File: 2025-1/sesion-de-clase-18/SoftProgNet/SoftProgModelo/Logistica/Ventas/OrdenVenta.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Security.Permissions;
using System.Text;
using System.Threading.Tasks;
using PUCP.Edu.Pe.SoftProg.Modelo.Clientes;
using PUCP.Edu.Pe.SoftProg.Modelo.Logistica.Ventas;
using PUCP.Edu.Pe.SoftProg.Modelo.RRHH;

namespace PUCP.Edu.Pe.SoftProg.Modelo.Logistica.Ventas {
    public class OrdenVenta : ModeloBase {
        public Cliente Cliente { get; set; }
        public Empleado Empleado { get; set; }
        public List<LineaOrdenVenta> LineasOrdenVenta { get; set; }
        public double Total {  get; set; }
        public DateTime FechaHora { get; set; }
    }
}

File: 2025-1/sesion-de-clase-20/SoftProg/SoftProgModelo/src/main/java/pe/edu/pucp/inf30/softprog/model/logistica/ventas/OrdenVenta.java
package pe.edu.pucp.inf30.softprog.model.logistica.ventas;

import java.util.Date;
import java.util.List;
import pe.edu.pucp.inf30.softprog.model.ModeloBase;
import pe.edu.pucp.inf30.softprog.model.clientes.Cliente;
import pe.edu.pucp.inf30.softprog.model.rrhh.Empleado;

/**
 *
 * @author eric
 */
public class OrdenVenta extends ModeloBase {
    private Cliente cliente;
    private Empleado empleado;
    private List<LineaOrdenVenta> lineasOrdenVenta;
    private double total;
    private Date fechaHora;
    
    public Cliente getCliente() {
        return cliente;
    }

    public void setCliente(Cliente cliente) {
        this.cliente = cliente;
    }

    public Empleado getEmpleado() {
        return empleado;
    }

    public void setEmpleado(Empleado empleado) {
        this.empleado = empleado;
    }

    public List<LineaOrdenVenta> getLineasOrdenVenta() {
        return lineasOrdenVenta;
    }

    public void setLineasOrdenVenta(List<LineaOrdenVenta> lineasOrdenVenta) {
        this.lineasOrdenVenta = lineasOrdenVenta;
    }

    public double getTotal() {
        return total;
    }

    public void setTotal(double total) {
        this.total = total;
    }

    public Date getFechaHora() {
        return fechaHora;
    }

    public void setFechaHora(Date fechaHora) {
        this.fechaHora = fechaHora;
    }
}